src/components/ServerStatus.js

```diff
@@ -26,7 +26,7 @@ const ServerStatus = (props) => {
   return (
     props.isOnline !== undefined && 
```

Comment:
Maybe 
`props.isOnline !== undefined &&  props.network && ...`

Reply:
I've updated to something similar but in a way that I think the code is more clear. Can you see if it's good?

Reply:
Yep